### PR TITLE
Make audio support optional

### PR DESCRIPTION
**Pull request by linkmauve**
This reduces the amount of dependencies of 16 if audio support isn’t required by the game.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,8 +15,9 @@ exclude = ["doc/*", "bors.toml", ".travis.yml", "test_data/*"]
 [lib]
 
 [features]
-default = ["opengl"]
+default = ["opengl", "audio"]
 opengl = ["gfx_device_gl", "gfx_window_glutin", "glutin"]
+audio = ["rodio"]
```
Comment (kvark) [inline Cargo.toml]: would you consider changing the feature name to simply "rodio"?

Comment (linkmauve) [reply Cargo.toml]: Sure, but that sounds less like something the user may understand, at first sight.

Comment (kvark) [reply Cargo.toml]: yeah, I take the point. Let's proceed then

**Comment by kvark** [comment]
It just occurred to me that we are missing CI checks for builds without "audio" feature

**Comment by linkmauve** [comment]
This might work, let’s wait for CI to run.

**Comment by linkmauve** [comment]
Turns out three doesn’t currently build when the opengl feature is absent, I force-enabled it in this commit. It would be useful to fix that eventually.

**Comment by linkmauve** [comment]
Seems to test just fine, thanks for the comment!

**Comment by kvark** [comment]
Amazing work, thank you!
bors r+

**Pull request merged**